### DISCUSSION TITLE
multiple PCA9685 expander selector

### DISCUSSION THREAD

**Discussion by advancedcodingenterprise** [ArminJo/ServoEasing#56] [General]
hello again. I have a quick question because the example don't clear this much for me.
So in my situation I have 2 PCA9685 expanders but how do I ignite movement for servos from chosen expander.
Example I want to move 3 servos from expander 1 and 2 servos from expander 2.
I know using `ServoEasing::ServoEasingArray[i]` is the way to go but in that case how would this array be filled with the expander 2 servos, are they adjestent to the index after the first expander or is it something else? Example:

- Expander 1 is stored in  ServoEasingArray keys 0 to 15
- Expander 2 is stored in ServoEasingArray keys 16 to 31

and if that is the case is that Array stored based on the sequence of the attach method I call?

**Comment by ArminJo** [comment]
Only the order of the attach() statements determines the position in the array.
See https://github.com/ArminJo/ServoEasing#handling-multiple-servos-with-the-internal-servoeasingarray

**Reply by advancedcodingenterprise** [reply]
so its like I suspected - servo 3 from expansion board 2 would be refered to as 18 (as its the 18th atachment from attach method)
